docs/topics/mta-cli-args.adoc

```diff
@@ -146,39 +172,40 @@ Specify the path to the directory to output the report information generated by
 --output <OUTPUT_REPORT_DIRECTORY>
 ----
 
-* The `--output` argument is mandatory. If omitted, an error is returned:
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
+* If the output directory exists, you will be prompted with the following (with a default of N).
```

Comment:
```suggestion
* If the output directory exists, you will be prompted with the following question with a default answer of `N`:
```
Is it really required to state the the N answer is the default one? I'd assume user should know such trivial things :-D But who know....Anything can happen